SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialLoadBalanceEnforcer.h

```diff
@@ -19,20 +19,31 @@ class SpatialLoadBalanceEnforcer
 	struct AclWriteAuthorityRequest
 	{
 		Worker_EntityId EntityId = 0;
-		FString OwningWorkerId;
+		PhysicalWorkerName OwningWorkerId;
+		WorkerRequirementSet ReadAcl;
+		WorkerRequirementSet ClientRequirementSet;
+		TArray<Worker_ComponentId> ComponentIds;
 	};
 
 	SpatialLoadBalanceEnforcer(const PhysicalWorkerName& InWorkerId, const USpatialStaticComponentView* InStaticComponentView, const SpatialVirtualWorkerTranslator* InVirtualWorkerTranslator);
 
-	void AuthorityChanged(const Worker_AuthorityChangeOp& AuthOp);
-	void QueueAclAssignmentRequest(const Worker_EntityId EntityId);
 
 	void OnAuthorityIntentComponentUpdated(const Worker_ComponentUpdateOp& Op);
+	void OnLoadBalancingComponentAdded(const Worker_AddComponentOp& Op);
+	void OnLoadBalancingComponentRemoved(const Worker_RemoveComponentOp& Op);
+	void OnEntityRemoved(const Worker_RemoveEntityOp& Op);
+	void OnAclAuthorityChanged(const Worker_AuthorityChangeOp& AuthOp);
+
+	void MaybeQueueAclAssignmentRequest(const Worker_EntityId EntityId);
 
 	TArray<AclWriteAuthorityRequest> ProcessQueuedAclAssignmentRequests();
 
 private:
 
+	void QueueAclAssignmentRequest(const Worker_EntityId EntityId);
+	bool AclAssignmentRequestIsQueued(const Worker_EntityId EntityId);
```

Comment:
```suggestion
	bool AclAssignmentRequestIsQueued(const Worker_EntityId EntityId) const;
```

Reply:
done

Reply:
oh wait only did the const

Reply:
I prefer boolean functions like this to be a statement rather than a question - if (statement) then... makes more sense to me